.yamato/shadergraph-osx-openglcore.yml

```diff
@@ -15,7 +15,7 @@ ShaderGraph_OSX_OpenGLCore_playmode_mono_apiNet2_Linear_2020.2:
       -  curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr --output TestProjects/ShaderGraph/utr
       -  chmod +x TestProjects/ShaderGraph/utr
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
-      -  cd TestProjects/ShaderGraph && unity-downloader-cli -u dff91d65251a9ab8c12af258ce88fee061d85a7d -c editor -c il2cpp  --wait --published-only
+      -  cd TestProjects/ShaderGraph && unity-downloader-cli -u 19f63eee530cf75bf6b9e470d2b27eedda41a780 -c editor -c il2cpp  --wait --published-only
```

Comment:
why is this one different?

Reply:
The revisions are different depending on platforms, it's probably because it's on OSX?
To get to the revisions used by this PR I used the file generated by tonight's update job: https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/ci%252Feditor-pinning/.yamato%252F_editor_pinning.yml%2523editor-pinning-merge-revisions-trunk-abv/4010178/job/pipeline.
--> Click the commit hash > Ctrl-F `_latest_editor_versions_2020.2.metafile` > this is the file I used